src/app/features/reminders/reminder-card/reminder-card.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import Status, { StatusDesc } from '../../../core/enums/status.enum';
+import { Reminder } from '../../../core/models/reminder.model';
+
+@Component({
+  selector: 'app-reminder-card',
+  templateUrl: './reminder-card.component.html',
+  styleUrls: ['./reminder-card.component.scss']
+})
+export class ReminderCardComponent implements OnInit {
+  @Input() reminder!: Reminder;
+  @Input() save!: () => void;
+  @Input() reminders: Reminder[];
+  @Output() remindersUpdate = new EventEmitter<Reminder[]>();
+
+  constructor() {
+    this.reminders = [];
+  }
+
+  ngOnInit(): void {
+    this.reminders = this.getReminders();
```

Comment:
Nope, don't do that... you may keep single responsibility to each component. This component is defined to receive the reminders as input, as other parameters, so you should move this to the parent component as well.

Reply:
I agree 100% with this, in the moment I came up with this solution, I didn't feel right to pass all the reminders as parameters, I don't feel like doing that request there as well.

I prefer to find a way to be able to use the parent component data in child component, without having to pass or request that data. 
But reviewing it now, it for sure would've not been that bad to just pass the reminders.